src/applications/personalization/profile360/components/ProfileView.jsx

```diff
@@ -185,6 +187,11 @@ class ProfileView extends React.Component {
     return (
       <div className="va-profile-wrapper row" style={{ marginBottom: 35 }}>
         <div className="usa-width-two-thirds medium-8 small-12 columns">
+          {/* Breadcrumbs */}
+          <Breadcrumbs className="medium-screen:vads-u-padding-y--2">
+            <a href="/">Home</a>
+            <a href="/">Your profile</a>
```

Comment:
The Breadcrumb component will ensure that this link is never clickable since it's always going to be the last crumb?

Reply:
We add `pointer-events:none` to the last element which prevents it from being clickable 😄 
![image](https://user-images.githubusercontent.com/14869324/81305185-4565ef80-903b-11ea-9e69-d44c5c9bfde5.png)